options.go

```diff
@@ -1,5 +1,37 @@
 package iavl
 
+import "sync/atomic"
+
+// Statisc about db runtime state
+type Statistics struct {
+	// Each time GetNode and GetFastNode operation hit cache
+	cacheHitCnt uint64
```

Comment:
We mainly have 2 types of nodes:
1. Regular (that you get with `GetNode(...)`
2. Fast (`GetFastNode(...)`

They each have their own cache (`nodeCache`, `fastNodeCache`):
https://github.com/cosmos/iavl/blob/640ad78af7f34486f4793b3e442c8d20c751daf0/nodedb.go#L79-L83

What do you think about having a separate counter for each cache?

Reply:
That is great. Done